docs/conf.py

```diff
@@ -195,6 +196,16 @@
 ]
 
 
+# -- Options for Sphinx AutoAPI output ----------------------------------------------
+
+# Paths (relative or absolute) to the source code that you wish
+# to generate your API documentation from.
+autoapi_dirs = ['../blueoil/']
+
+# A list of patterns to ignore when finding files.
+autoapi_ignore = ['*migrations*', '*/configs/*', '*/templates/*']
```

Comment:
Can we specify more concrete paths for `configs` and `templates`? `blueoil/configs/*` or something.

Reply:
I think the Sphinx-autoapi ignore format is an absolute path. Thus I cannot ignore the `*` prefix.
I add `/blueoil/` to them to `*/blueoil/configs/*`, `*/blueoil/templates/*`.

https://sphinx-autoapi.readthedocs.io/en/latest/reference/config.html#confval-autoapi_ignore

Reply:
Thanks for checking. I guess we need to include the root dir like `../blueoil/configs/*`. If this doesn't work, it's fine as it is 👍 

https://github.com/readthedocs/sphinx-autoapi/blob/9397c0709f086f9a656bdc5bc535134b4084349b/autoapi/mappers/base.py#L225

Reply:
I tried `../blueoil/configs/*` before, it does not work, unfortunately.

Reply:
OK! Thanks.